onnx/defs/tensor/defs.cc

```diff
@@ -251,27 +254,27 @@ Otherwise, the tensor is split to equal sized parts.
 )DOC")
     .TypeAndShapeInferenceFunction([](InferenceContext& ctx) {
       for (int i = 0; i < static_cast<int>(ctx.getNumOutputs()); ++i) {
-        propagateElemTypeFromInputToOutput(ctx, 0, i);
+        ONNX_RETURN_IF_ERROR(propagateElemTypeFromInputToOutput(ctx, 0, i));
       }
 
-      if (!hasNInputShapes(ctx, 1)) {
-        return;
-      }
+	  if (hasNInputShapes(ctx, 1)) {
```

Comment:
Missing !

Reply:
Thank you so much for the catch and fix!